### ISSUE TITLE
Ne sauvegarder le profil que lorsque l'utilisateur·rice l'a demandé

### ISSUE THREAD

**Issue by DavidLibeau** [LAB-MI/deplacement-codiv-19#6]
Le formulaire sauvegarde ses données dans le stockage local du navigateur pour que l'utilisateur·rice n'ait pas a renseigner à nouveau l'intégralité des informations lorsqu'iel souhaite générer un pdf. Cette fonctionnalité est très pratique. Cependant, je pense qu'il serait souhaitable de demander l'avis de l'utilisateur·rice avant d'enregistrer toute donnée.

Ainsi, il faudrait que [la fonction `saveProfile()`](https://github.com/LAB-MI/deplacement-codiv-19/blob/master/certificate.js#L52) ne soit exécutée que lorsque qu'une case à cocher soit validée.

Par exemple :
- [ ] Enregistrer mes informations dans mon navigateur web pour ne pas à avoir les renseigner la prochaine fois

**Comment by JohnXLivingston** [comment]
D'après ce que j'ai constaté, il n'est pas sauvegardé à cette fin. En réalité, ce qui est stocké n'est jamais lu après coup (en tout cas dans la version qui était en prod ce 6 avril vers 17h). La fonction getProfile n'est appelée que dans le submit du formulaire, juste après le save...
Je pense que c'est un reste de code, parce qu'ils avaient effectivement dû prévoir de restaurer les données, mais que finalement ils ne l'ont pas fait.

J'ai signalé le bug (sur twitter), et on m'a dit qu'il serait corrigé rapidement. 

À noter qu'il pose - à mon sens - un grave problème de sécurité. Si vous générez le pdf sur  un terminal qui n'est pas le votre, les données restent. Si vous le faite sur votre smartphone et qu'on vout le vole, on peut vous voler votre état civil. Etc.

**Comment by DavidLibeau** [comment]
Effectivement, je n'ai pas eu le temps de faire une issue sur ce sujet mais il semblerait que les données soient stockées mais jamais utilisées.
